packages/genesys-spark-components/src/components/stable/gux-table/gux-table.tsx

```diff
@@ -42,9 +42,12 @@ export class GuxTable {
   root: HTMLElement;
 
   private resizeObserver: ResizeObserver;
-  private slotObserver: MutationObserver = new MutationObserver(() =>
-    forceUpdate(this)
-  );
+  private slotObserver: MutationObserver = new MutationObserver(() => {
+    if (this.selectAllCheckbox) {
```

Comment:
I am not sure this if guard is needed. The `updateSelectAllBoxState` method that is eventually calls has a guard against the select all checkbox not existing already.